src/MIDebugEngine/Engine.Impl/EngineUtils.cs

```diff
@@ -109,6 +109,33 @@ internal static bool ProcIdEquals(AD_PROCESS_ID pid1, AD_PROCESS_ID pid2)
             }
         }
 
+
+        /// <summary>
+        /// This allows console commands to be sent through the eval channel via a '-exec ' or '`' preface
+        /// </summary>
+        /// <param name="command">raw command</param>
+        /// <param name="strippedCommand">command stripped of the preface ('-exec ' or '`')</param>
+        /// <returns>true if it is a console command</returns>
+        internal static bool IsConsoleExecCmd(string command, out string prefix, out string strippedCommand)
+        {
+            prefix = string.Empty;
+            strippedCommand = string.Empty;
+            string execCommandString = "-exec ";
+            if (command.StartsWith(execCommandString, StringComparison.Ordinal))
+            {
+                prefix = execCommandString;
+                strippedCommand = command.Substring(execCommandString.Length);
+                return true;
+            }
+            else if (command[0] == '`')
+            {
+                prefix = "`";
+                strippedCommand = command.Substring(1).TrimStart(); // remove spaces if any
```

Comment:
>strippedCommand [](http://example.com/codeflow?start=16&length=15)

Should this check that there is something left after the whitespace?

Reply:
Or does GDB handle that okay?

Reply:
GDB seems to be ok with extranous whitespace:
![image](https://user-images.githubusercontent.com/3953714/135545933-53a05157-adba-4aef-9afe-a10c5fc09b33.png)

Reply:
Sorry, I mean is GDB happy if you ask for complete after `-exec` (no command typed yet)?

Reply:
Ah, VS Code does not seem to send an autocomplete on a string ending in whitespace or `. We kind of lucked out there. 

But to be extra careful, do we want to do `TrimEnd`?

Reply:
What about `-exec`?

Reply:
`-exec` does not match `-exec ` (with a space) 😅

Reply:
Ahh... Cool.